app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/ActualWalkingFragment.java

```diff
@@ -0,0 +1,254 @@
+package org.pesmypetcare.mypetcare.activities.fragments.walks;
+
+import android.content.Context;
+import android.location.Location;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.Fragment;
+
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationCallback;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationResult;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.JointType;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Polyline;
+import com.google.android.gms.maps.model.PolylineOptions;
+import com.google.android.gms.maps.model.RoundCap;
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.textfield.TextInputLayout;
+
+import org.pesmypetcare.mypetcare.R;
+import org.pesmypetcare.mypetcare.activities.MainActivity;
+import org.pesmypetcare.mypetcare.activities.fragments.infopet.InfoPetFragment;
+import org.pesmypetcare.mypetcare.activities.fragments.infopet.InfoPetHealthFragment;
+import org.pesmypetcare.mypetcare.databinding.FragmentActualWalkingBinding;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Albert Pinto
+ */
+public class ActualWalkingFragment extends Fragment implements OnMapReadyCallback, ActualWalkingCommunication {
+    private static final float ZOOM = 16.0f;
+    private static final int REQUEST_INTERVAL = 10000;
+    private static final int FASTEST_INTERVAL_REQUEST = 5000;
+    private static Context context;
+    private static LayoutInflater inflater;
+    private static String errorMessage;
+    private GoogleMap googleMap;
+    private static FusedLocationProviderClient fusedLocationProviderClient;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5636997)